Holovibes/sources/API.cc

```diff
@@ -403,30 +403,14 @@ void set_view_mode(const ImgType type)
 
 void update_batch_size(const uint batch_size)
 {
-    if (batch_size == get_batch_size())
+    if (UserInterfaceDescriptor::instance().import_type_ == ImportType::None || get_batch_size() == batch_size)
```

Comment:
Pk ?

Reply:
It's moved code from the UI to the API. But in the future there must be 0 call to UserInterfaceDedcriptor in the API and WE should create a gui namespace where all functions reside.